creator-node/src/serviceRegistry.js

```diff
@@ -15,26 +18,31 @@ const config = require('./config')
  *  - `ipfs`: IPFS Client
  *  - `ipfsLatest`: IPFS Client, latest version
  *  - `blackListManager`: responsible for handling blacklisted content
- *  - `audiusLibs`: an instance of Audius Libs
+ *  - `libs`: an instance of Audius Libs
+ *  - `monitoringQueue`: recurring job to monitor node state & performance metrics
+ *  - `nodeConfig`: exposes config object
+ *  - `snapbackSM`: SnapbackStateMachine is responsible for recurring sync and reconfig operations
+ *  - `URSMRegistrationManager`: registers node on L2 URSM contract, no-ops afterward
  *
  * `initServices` must be called prior to consuming services from the registry.
  */
 class ServiceRegistry {
   constructor () {
+    this.nodeConfig = config
```

Comment:
what is the benefit of assigning to `this.nodeConfig` instead of referencing the convict object directly? if its just a reference its fine but i want to ensure we don't make a copy and end up with updates not propagating (might be overthinking this)

Reply:
It should be a reference to the config module so updates shouldn't be a problem. But I kind of agree with @hareeshnagaraj here that we probably don't need this? I can see an argument for both adding here or not

Reply:
ya it is just a reference, same as requiring individually. am thinking of this as a configService so passing around similarly. also per my earlier comment, it makes it much much easier to unit test code